lib/Sema/TypeCheckType.cpp

```diff
@@ -4232,6 +4221,7 @@ TypeResolver::resolveDeclRefTypeRepr(DeclRefTypeRepr *repr,
 NeverNullType
 TypeResolver::resolveOwnershipTypeRepr(OwnershipTypeRepr *repr,
                                        TypeResolutionOptions options) {
+  auto ownershipRepr = dyn_cast<OwnershipTypeRepr>(repr);
```

Comment:
could just rename the parameter

Reply:
I think this was just an artifact from merging my old branch, where we still worked on a SpecifierTypeRepr at this point. Thanks!